config/manager/kustomization.yaml

```diff
@@ -2,6 +2,7 @@ resources:
 - manager.yaml
 - metrics_service.yaml
 - console-plugin-images.yaml
+- controller_manager_config.yaml
```

Comment:
maybe we can remove this file, it was part of the update of controller runtime: https://github.com/Kuadrant/kuadrant-operator/pull/246